### ISSUE TITLE
Error while building new Angular-9 project created using Angular-cli with bazel

### ISSUE THREAD

**Issue by preetham1290** [angular/angular#36789]
# 🐞 bug report

### Affected Package

 The issue is caused by package @angular/bazel

### Is this a regression?

 No, am new to this bazel.

### Description

 Did exactly as provided in `https://angular.io/guide/bazel` but getting error when i try to build or run. 
```
 # ng new --collection=@angular/bazel
? What name would you like to use for the new workspace and initial project? bzl-test
? Would you like to add Angular routing? No
? Which stylesheet format would you like to use? SCSS   [ http://sass-lang.com/documentation/file.SASS_REFERENCE.html#syntax ]
CREATE bzl-test/README.md (1024 bytes)
.
.
CREATE bzl-test/angular.json.bak (3804 bytes)
✔ Packages installed successfully.
            Successfully initialized git
```
After few research i even deleted `node_modules` and did npm install again but still same error.

## 🔬 Minimal Reproduction

 https://github.com/preetham1290/bzl-test

## 🔥 Exception or Error
<pre><code>
# ng serve 
iBazel [6:03PM]: Error getting Bazel info fork/exec /home/angular/ng9/bzl-test/node_modules/@bazel/bazelisk/bazelisk-linux_amd64: no such file or directory
iBazel [6:03PM]: Querying for files to watch...
iBazel [6:03PM]: Bazel query failed: fork/exec /home/angular/ng9/bzl-test/node_modules/@bazel/bazelisk/bazelisk-linux_amd64: no such file or directory
iBazel [6:03PM]: Bazel query failed: fork/exec /home/angular/ng9/bzl-test/node_modules/@bazel/bazelisk/bazelisk-linux_amd64: no such file or directory
iBazel [6:03PM]: Running //src:devserver
iBazel [6:03PM]: Error running Bazel fork/exec /home/angular/ng9/bzl-test/node_modules/@bazel/bazelisk/bazelisk-linux_amd64: no such file or directory
/home/angular/ng9/bzl-test/node_modules/@bazel/ibazel/bin/linux_amd64/ibazel failed with code 4.
</code></pre>

Then tried to build using both `ng build` ang `ng build --prod`
```
# ng build 

Starting local Bazel server and connecting to it...
INFO: Analyzed target //src:prodapp (203 packages loaded, 13948 targets configured).
INFO: Found 1 target...
INFO: Deleting stale sandbox base /root/.cache/bazel/_bazel_root/d7f0e0363388a328c2388dde618ec408/sandbox
ERROR: /home/angular/ng9/bzl-test/src/BUILD.bazel:27:1: Compiling Angular templates (Ivy - prodmode) //src:src failed (Exit 1)
external/npm/node_modules/@angular/platform-browser/platform-browser.d.ts:44:22 - error NG6002: Appears in the NgModule.imports of AppModule, but could not be resolved to an NgModule class.

This likely means that the library (@angular/platform-browser) which declares BrowserModule has not been processed correctly by ngcc, or is not compatible with Angular Ivy. Check if a newer version of the library is available, and update if so. Also consider checking with the library's authors to see if the library is expected to be compatible with Ivy.

44 export declare class BrowserModule {
                        ~~~~~~~~~~~~~

Target //src:prodapp failed to build
Use --verbose_failures to see the command lines of failed build steps.
ERROR: /home/angular/ng9/bzl-test/src/BUILD.bazel:90:1 Compiling Angular templates (Ivy - prodmode) //src:src failed (Exit 1)
INFO: Elapsed time: 18.123s, Critical Path: 8.67s
INFO: 1 process: 1 linux-sandbox.
FAILED: Build did NOT complete successfully
/home/angular/ng9/bzl-test/node_modules/@bazel/bazel/node_modules/@bazel/bazel-linux_x64/bazel-2.1.0-linux-x86_64 failed with code 1.
```

## 🌍  Your Environment

**Angular Version:**
<pre><code>
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 9.1.3
Node: 10.15.0
OS: linux x64

Angular: 
... 
Ivy Workspace: 

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.901.3
@angular-devkit/core         9.1.3
@angular-devkit/schematics   9.1.3
@angular/bazel               9.1.3
@angular/cli                 9.1.3
@angular/compiler            9.1.3
@angular/compiler-cli        9.1.3
@bazel/typescript            1.6.0
@schematics/angular          9.1.3
@schematics/update           0.901.3
rxjs                         6.5.4
typescript                   3.8.3
</code></pre>

**Anything else relevant?**

**Comment by petebacondarwin** [comment]
I cloned the git repository and ran:

```
npm i
```

This ran the `ngcc` postinstall step and processed all the dependencies successfully.

Running `ng build` and `ng build --prod` succeeded too.

I see the same original error for `ng serve`.

I suspect that the secondary errors related to ngcc were due to a corrupt node_modules folder and is a red-herring.

Re-assigning it as an issue with `bazel`.
